rpms/s3/buildrpm.sh

```diff
@@ -107,6 +107,7 @@ elif ! [ -z "${PATH_SRC}" ]; then
 fi
 
 cd ~/rpmbuild/SOURCES/
+rm -f ./cortx-s3server-${S3_VERSION}-git${GIT_VER}/scripts/s3-sanity-test.sh
```

Comment:
why we need this change ? @sachitanands !!!

Reply:
So that the file does not get packaged

Reply:
@amitwac2608  yes.. moving forward we are pushing all the test related scripts to this new test rpm (cortx-s3server-test.rpm)
This script needs s3iamcli and s3cmd configured as pre-requisite hence we moved it test rpm

Reply:
Done